pkg/prompts/prompts.go

```diff
@@ -155,23 +139,21 @@ func validateBiggerThanZero(input string) error {
 // and `CaptureAddress` is supported) until the user cancels or
 // chooses `Done`. It does also offer an optional `info` to print
 // (if provided) and a preview. Items can also be removed.
-func (r *realPrompter) CaptureListDecision(
+func CaptureListDecision[T comparable](
```

Comment:
Why did you take this out of the interface?

Reply:
https://github.com/ava-labs/avalanche-cli/issues/258